src/PlutusData.purs

```diff
@@ -0,0 +1,18 @@
+module PlutusData
```

Comment:
Could this go into `Helpers`? If so I personally think that would be better than a whole new module for this (unless there's some circular dependency issue with that too)

Reply:
Will try add it in.

Reply:
Hmm, that caused some horrible cyclic deps.

Reply:
Going to leave it for now.

Reply:
OK, no problem. There's no way around the extra module then